content/altinn-studio/getting-started/access-management-apps/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Tilgangsstyring i Altinn Apps
+linktitle: Tilgangsstyring i Altinn Apps
+description: Hvordan bestille tilganger til Altinn Apps
+toc: true
+weight: 200
+
+---
+
+## Tilgangsstyring i Apps
```

Comment:
Better if changed to "Tilgang til logger og hemmeligheter"? Currently almost the same as toplevel header.